test/Microsoft.DotNet.ProjectModel.Loader.Tests/Microsoft.DotNet.ProjectModel.Loader.Tests.xproj

```diff
@@ -13,9 +12,11 @@
     <OutputPath Condition="'$(OutputPath)'=='' ">.\bin\</OutputPath>
     <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
   </PropertyGroup>
-
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```

Comment:
I usually hack them out.  But yes, it is annoying.  Good thing .xproj isn't sticking around for long.